resultset_rows.go

```diff
@@ -50,8 +67,12 @@ func init() {
 	}
 }
 
-type rows struct {
-	conn *conn
+type ResultSetConn interface {
+	getStatement(ctx context.Context, statementID uuid.UUID, partitionID int32) (rs *apiv2.ResultSet, err error)
```

Comment:
Should this be a public function?

Reply:
no, it is only used internally in this package. It abstracts away the differences between conn and dpconn so that both can be used by the resultset_rows